cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,31 @@
-describe('', () => {
-  before(() => {
-
+describe('Article flow', () => {
+  let user;
+  let article;
+  beforeEach(() => {
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+    });
+    cy.task('generateArticle').then((generateArticle) => {
+      article = generateArticle;
+    });
   });
-
-  it('', () => {
-
+  it('the user should be able to create an article', () => {
+    cy.login(user.email, user.username, user.password);
+    cy.createArticle(article.title, article.description, article.body);
```

Comment:
This is UI test and you need to create article via UI. This method should be used in the "delete article" test as precondition
```suggestion
    cy.createArticle(article.title, article.description, article.body);
```